cmd/web/config/config_test.go

```diff
@@ -471,3 +489,66 @@ func TestValidatorTypes_AsStringSlice(t *testing.T) {
 		}
 	})
 }
+
+func TestConfig_GetSensored(t *testing.T) {
```

Comment:
In the current situation, even if `GetSensored` did nothing, the test would still pass. I think a better approach would be to set the three values to something else than `mask` and I suppose we should move `mask` out of the function and define it on package scope, so that we can rely on its value, without needing to define it here as well.

So instead of:
```go
func (c Config) GetSensored() Config {
	const mask = "**masked**"
  ..
}
```

We should introduce:
```go
const (
  valueMask = "**masked**"
)
```

Which you can use is both `GetSensored()` and in your unit-tests, making it more robust.